sdk/storage/Azure.Storage.DataMovement/src/StorageTransferManager.cs

```diff
@@ -0,0 +1,134 @@
+        /// TODO: remove suppresion
+#pragma warning disable CS1998 // Async method lacks 'await' operators and will run synchronously
+        public async Task<StorageTransferResults> ScheduleDownloadJobAsync(
+#pragma warning restore CS1998 // Async method lacks 'await' operators and will run synchronously
+            BlobClient sourceClient,
+            string destinationLocalPath,
+            StorageTransferOptions transferOptions = default,
+            IProgress<StorageTransferStatus> progressTracker = default,
+            CancellationToken token = default)
+        {
+            //TODO: if check the local path exists and not a directory
+            // or we can go and check at the start of the job, to prevent
+            // having to check the existence of the path twice.
+            BlobTransferJob transferJob = new BlobTransferJob(sourceClient, destinationLocalPath, transferOptions, progressTracker, token);
+            _jobsToProcess.Enqueue(transferJob);
+
+            // TODO; remove stub
+            return new StorageTransferResults();
+        }
+
+        /// <summary>
+        /// Add upload job to perform.
+        /// </summary>
+        /// TODO: remove suppression
+#pragma warning disable CS1998 // Async method lacks 'await' operators and will run synchronously
+        public async Task<StorageTransferResults> ScheduleUploadDirectoryJobAsync(
+#pragma warning restore CS1998 // Async method lacks 'await' operators and will run synchronously
+            string sourceLocalPath,
+            BlobDirectoryClient destinationClient,
+            StorageTransferOptions transferOptions = default,
+            BlobUploadOptions uploadOptions = default,
+            IProgress<StorageTransferStatus> progressTracker = default,
+            CancellationToken token = default)
+        {
+            //TODO: if check the local path exists and not a directory
+            // or we can go and check at the start of the job, to prevent
+            // having to check the existence of the path twice.
+            BlobDirectoryTransferJob transferJob = new BlobDirectoryTransferJob(sourceLocalPath, destinationClient, transferOptions, uploadOptions, progressTracker, token);
+            _toScanQueue.Enqueue(transferJob);
+
+            // TODO; remove stub
+            return new StorageTransferResults();
+        }
+
+        /// <summary>
+        /// Pause transfers.
+        /// </summary>
+        /// TODO: Returns actual object, or at least in a designated log
+        /// file we have a place where people can continue transfers
+        public static void PauseTransfer()
+        {
+        }
+
+        /// <summary>
+        /// Cancel Transfers
+        /// </summary>
+        public static void CancelTransfer()
+        {
+            // This would remove all transfers from the queue and not log the current progress
+            // to the file. Maybe we would also remove the file too as a part of cleanup.
+        }
```

Comment:
The docs say it's plural but API is singular.

Should these be static? I.e. are they going to pause/cancel transfers on all instances of StorageTransferManager?
Maybe we'll eventually need instance level and global level methods like this. But should we start with instance level?